packages/clay-link/src/index.tsx

```diff
@@ -33,22 +33,37 @@ interface IProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
 	/**
 	 * Determines how the link is displayed.
 	 */
-	displayType?: 'primary' | 'secondary' | 'unstyled';
+	displayType?: 'danger' | 'primary' | 'secondary' | 'tertiary' | 'unstyled';
 
 	/**
-	 * Flag to indicate if link should be monospaced.
+	 * Sets the text size based on a number scale.
+	 */
+	fontSize?: Number;
+
+	/**
+	 * Flag to indicate if the link should be monospaced.
 	 */
 	monospaced?: boolean;
 
 	/**
-	 * Flag to indicate if link need have an outline.
+	 * Flag to indicate if the link should use the outlined style.
 	 */
 	outline?: boolean;
 
 	/**
-	 * Indicates button should be a small variant.
+	 * Indicates whether the button should use the small variant.
 	 */
 	small?: boolean;
+
+	/**
+	 * Determines the font-weight of the link.
+	 */
+	thickness?: 'normal' | 'semi-bold';
+
+	/**
+	 * Indicates if the text should be underlined
+	 */
+	underline?: boolean;
```

Comment:
I think the underline will be required for all non-button links.

Reply:
@matuzalemsteles it doesn't seem like it will always be required according to https://www.figma.com/file/r0oZ3ORLODdHmvvcyCASoQ/Links?node-id=1%3A56&t=C7qv8XUFgWbgUbRk-1. What if we default to true?

Reply:
Yeah, I think true by default is a good idea, `underline` should add to all links but except for some use cases. Figma's example is for some cases that won't need an underline, I think this Slack thread has more information about that https://liferay.slack.com/archives/C04A7NLM7L1/p1672744222205579